### PR TITLE
[Localization] Localized Daily Run Scoreboard

### PR DESCRIPTION
**Pull request by josericardo-fo**
## What are the changes?

Localized the Daily Run Scoreboard

## Why am I doing these changes?

It needs localization

## What did change?

The labels of the columns in the scoreborad

## Before (ptBR)
image

## After (ptBR)
(no assets yet)

## How to test the changes?

Run a server and check the scoreboards

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [ ] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)
    I got a bunch of errors that were not related to my changes
- [X] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/de/menu.ts [hunk 1]
```diff
@@ -44,6 +44,9 @@ export const menu: SimpleTranslationEntries = {
   "dailyRankings": "Tägliche Rangliste",
   "weeklyRankings": "Wöchentliche Rangliste",
   "noRankings": "Keine Rangliste",
+  "usernameScoreboard": "Username",
+  "score": "Score",
+  "wave": "Wave",
```
Comment (flx-sta) [inline src/locales/de/menu.ts:49]:
```suggestion
  "usernameScoreboard": "Benutzername",
  "score": "Punktzahl",
  "wave": "Welle",
```

File: src/ui/daily-run-scoreboard.ts [hunk 2]
```diff
@@ -154,7 +154,7 @@ export class DailyRunScoreboard extends Phaser.GameObjects.Container {
       return entryContainer;
     };
 
-    this.rankingsContainer.add(getEntry("#", "Username", "Score", "Wave"));
+    this.rankingsContainer.add(getEntry("#", i18next.t("menu:usernameScoreboard"), i18next.t("menu:score"), i18next.t("menu:wave")));
```
Comment (flx-sta) [inline src/ui/daily-run-scoreboard.ts:157]: Idk if there is a language where they don't use `#` for the number abbrv. 
Should it be localized too?

**Comment by flx-sta** [comment]
image

seems to work.
There is some comments open from me.
I see a problem with the `score` word in many languages as not every language has such a short word for it.

File: src/ui/daily-run-scoreboard.ts (hunk 2 above)
Comment (josericardo-fo) [reply src/ui/daily-run-scoreboard.ts]: localized

**Comment by josericardo-fo** [comment]
> image
> seems to work. There is some comments open from me. I see a problem with the `score` word in many languages as not every language has such a short word for it.

Do you think that this fix needs to be in this PR?

**Comment by flx-sta** [comment]
I don't necessarily think so, but I also think this looks super ugly if not :D
I'd get some devs opinion on it

**Review by EnochG1 — CHANGES_REQUESTED** [review]
I suggest korean locales :)

File: src/locales/de/menu.ts [hunk 3]
```diff
@@ -44,6 +44,10 @@ export const menu: SimpleTranslationEntries = {
   "dailyRankings": "Tägliche Rangliste",
   "weeklyRankings": "Wöchentliche Rangliste",
   "noRankings": "Keine Rangliste",
+  "positionIcon": "#",
+  "usernameScoreboard": "Benutzername",
+  "score": "Punkte",
```
Comment (flx-sta) [inline src/locales/de/menu.ts:49]:
```suggestion
  "score": "Pkt.",
```

**Review by EnochG1 — APPROVED** [review]

**Review by CodeTappert — APPROVED** [review]
Looks good.

**Review by CodeTappert — APPROVED** [review]

**Review by XGSleepWalker — APPROVED** [review]

**Pull request merged**